service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/workspace/CloneGcpWorkspaceFlight.java

```diff
@@ -28,7 +28,10 @@ public CloneGcpWorkspaceFlight(FlightMap inputParameters, Object applicationCont
         RetryRules.cloud());
     addStep(new AwaitCreateGcpContextFlightStep(), RetryRules.cloudLongRunning());
 
-    addStep(new ClonePolicyAttributesStep(flightBeanBag.getTpsApiDispatch()), RetryRules.cloud());
+    addStep(
```

Comment:
Might be simpler to put the feature check here and not put the step in the flight if TPS is not enabled.

That would also prevent "flip-flop" if the flight started with the feature disabled and, after a failover, tried to undo with the feature enabled.

Reply:
Makes sense, I'll move it here.